pyomo/opt/tests/base/test_ampl.py

```diff
@@ -83,7 +87,11 @@ def test3_write_lp(self):
                 self.fail("Unexpected ConverterError - glpsol is enabled "
                           "but not available: '%s'" % str(err))
             return
-        self.assertFileEqualsBaseline(currdir+'test3.lp', currdir+'test3.baseline.lp', filter=filter, tolerance=1e-6)
+        with open(currdir+'test3.lp', 'r') as run, \
+            open(currdir+'test3.baseline.lp', 'r') as baseline:
+                for line1, line2 in zip(run, baseline):
```

Comment:
This should probably use `itertools.zip_longest` so that differing length files cause errors.